src/api/readme.md

```diff
@@ -38,16 +38,18 @@ docker-compose up --build
 This will (re)build all the services, pull any images that are necessary for dependent services, and start all the services
 together. You can then access the services via the api hostname (see `API_HOST` in your .env file). For example:
 
-http://api.telescope.localhost/v1/image/gallery
+<http://api.telescope.localhost/v1/image/gallery>
```

Comment:
Why this change?

Reply:
Yikes, it seems when I updated the README and saved, it added those angle brackets together. That's something with the prettier settings iirc. I'll have to check on this